src/transform/transform.ts

```diff
@@ -465,6 +465,18 @@ function createResponseType(codeModel: CodeModel, group: OperationGroup, op: Ope
         prop.language.go!.fromHeader = item.value.header;
         (<Array<Property>>object.language.go!.properties).push(prop);
       }
+      const firstResp = <SchemaResponse>op.responses![0];
+      if (isLROOperation(op)) {
```

Comment:
Would this actually be hit?  Seems like line 428 would prevent this case from ever getting hit.

Reply:
I think you're right, I thought there would be a case where we could reach this without having added the LRO related fields. But on second look I dont think so. I'll remove this change.